### DISCUSSION TITLE
Enabling resolve button for code reviews

### DISCUSSION THREAD

**Discussion by ivanzlenko** [apache/ozone#7352] [General]
Is there a way to enable Resolve button for comments in Code reviews? 
It is a very helpful tool to track what comments have been handled by an author and if a reviewer agrees with changes.

**Comment by adoroszlai** [comment]
> You can resolve a conversation in a pull request if you opened the pull request or if you have write access to the repository where the pull request was opened.

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#resolving-conversations

**Reply by ivanzlenko** [reply]
@adoroszlai 
>  or if you have write access to the repository where the pull request was opened.

Ah I see, my initial thoughts was that it could be managed on a project level. But looks like it is GitHub-wide. Thanks! 
Will nudge authors of reviews then from now on :)
